devito/ops/operator.py

```diff
@@ -83,7 +85,7 @@ def _specialize_iet(self, iet, **kwargs):
             have the same number of dimensions"
 
         self._headers.append(namespace['ops_define_dimension'](dims[0]))
-        self._includes.append('stdio.h')
+        self._includes += ['stdio.h', 'ops_seq.h']
```

Comment:
extend

Reply:
this was nitpicking, but do you know why is it conceptually more sound?

Reply:
I found that INPLACE_ADD fails for numpy array.